### ISSUE TITLE
`departures`: work around broken `direction` field

### ISSUE THREAD

**Issue by derhuerst** [derhuerst/vbb-hafas#11]
``` javascript
client.departures 9023201,
    products:
        suburban: true
        subway: false
        bus: false
        regional: false
        express: false
    results: 2,
    when: new Date 'August 7, 2015 00:48:00 GMT+0200'
```

``` javascript
[
    {
        stop: 9023201,
        type: 'suburban',
        direction: 'S Strausberg Bhf, in Ostbahnhof bitte umsteigen',   // WAT
        when: Fri Aug 07 2015 00:49:00 GMT+0200 (CEST),
        line: 'S5'
    }, {
        stop: 9023201,
        type: 'suburban',
        direction: 'S Potsdam Hbf, in Wannsee bitte umsteigen',   // WAT
        when: Fri Aug 07 2015 00:52:00 GMT+0200 (CEST),
        line: 'S7',
        realtime: Fri Aug 07 2015 00:52:00 GMT+0200 (CEST)
    }
]
```

**Comment by derhuerst** [comment]
FYI there's now a workaround available as of `vbb-hafas@2.2.0`. [Each departure will have a `ref` property, which you can use to get the journey of its vehicle to get the last stop.](https://github.com/derhuerst/vbb-hafas/blob/master/docs/departures.md#response)

**Comment by derhuerst** [comment]
[`vbb-departures-in-direction`](https://github.com/derhuerst/vbb-departures-in-direction#vbb-departures-in-direction) does this now. [It is also exposed in the `vbb.transport.rest` API](https://github.com/derhuerst/vbb-rest/blob/556c370499b045a895a04273aa73a1ac5eb0660f/docs/index.md#get-stationsiddepartures).
